qemu/tests/cfg/sr_iov.cfg

```diff
@@ -6,7 +6,21 @@
     variants:
         - vf_boot:
             type = boot
-            pci_assignable_nic1 = vf
+            nics = "nic1 nic2"
+            pci_assignable_nic1 = no
+            nic_model_nic1 = virtio
+            pci_assignable_nic2 = vf
+            only x86_64
```

Comment:
I put it here because ppc team doesn't test the following four cards. Maybe we could submit another patch once ppc test is needed. Does it make any sense to you ?